README.md

```diff
@@ -16,11 +16,57 @@ To install, simply add it to your `composer.json` file:
 $ composer require myonlinestore/omnipay-klarna-checkout
 ```
 
+## Initialization
+
+```php
+$gateway = Omnipay::create('\MyOnlineStore\Omnipay\KlarnaCheckout\Gateway');
```

Comment:
I'd recommend direct creation rather than using the facade. Makes it easier to see what is actually executed

```suggestion
use MyOnlineStore\Omnipay\KlarnaCheckout\Gateway as Gateway;

$gateway = new Gateway(/* $httpClient, $httpRequest */);
```

Reply:
Valid point! However I suggest providing examples of both, since the main Omnipay package uses the facade approach in their example. It makes it more consistent and in line with their attempt to provide a standardized API regardless of provider.

Reply:
True, good for me :+1: